bindings/python/test/trajectory/test_sequence.py

```diff
@@ -318,6 +318,22 @@ def sequence(
     )
 
 
+@pytest.fixture
+def segment_solution(
+    dynamics: list,
+    state: State,
+):
+    return Segment.Solution(
+        name="A Segment",
```

Comment:
```suggestion
        name="A Segment Solution",
```

nit: Just checking, is this missing?